stacks/environment-aws/lead.tf

```diff
@@ -88,31 +88,51 @@ data "aws_ssm_parameter" "prometheus_slack_webhook_url" {
+}
+
+data "aws_ssm_parameter" "google_identity_provider_client_secret" {
+  count = var.enable_google_login ? 1 : 0
+  name  = "/${var.cluster}/google_identity_provider_client_secret"
+}
+
+data "aws_ssm_parameter" "test_user_password" {
+  count = var.enable_test_user ? 1 : 0
+  name  = "/${var.cluster}/test_user_password"
+}
+
 
 module "toolchain" {
-  source                     = "../../modules/lead/toolchain"
-  root_zone_name             = var.root_zone_name
-  cluster                    = module.eks.cluster_id
-  namespace                  = var.toolchain_namespace
-  image_whitelist            = var.image_whitelist
-  elb_security_group_id      = module.eks.aws_security_group_elb.id
-  artifactory_license        = data.aws_ssm_parameter.artifactory_license.value
-  keycloak_admin_password    = data.aws_ssm_parameter.keycloak_admin_password.value
-  keycloak_postgres_password = data.aws_ssm_parameter.keycloak_postgres_password.value
-  enable_istio               = var.enable_istio
-  enable_artifactory         = var.enable_artifactory
-  enable_gitlab              = var.enable_gitlab
-  enable_keycloak            = var.enable_keycloak
-  enable_mailhog             = var.enable_mailhog
-  enable_sonarqube           = var.enable_sonarqube
-  enable_xray                = var.enable_xray
-  enable_grafeas             = var.enable_grafeas
-  enable_harbor              = var.enable_harbor
-  issuer_name                = module.cluster_issuer.issuer_name
-  issuer_kind                = module.cluster_issuer.issuer_kind
-  crd_waiter                 = module.infrastructure.crd_waiter
-  grafeas_version            = var.grafeas_version
-  k8s_storage_class          = var.k8s_storage_class
+  source                                 = "../../modules/lead/toolchain"
+  root_zone_name                         = var.root_zone_name
+  cluster                                = module.eks.cluster_id
+  namespace                              = var.toolchain_namespace
+  image_whitelist                        = var.image_whitelist
+  elb_security_group_id                  = module.eks.aws_security_group_elb.id
+  artifactory_license                    = data.aws_ssm_parameter.artifactory_license.value
+  keycloak_admin_password                = data.aws_ssm_parameter.keycloak_admin_password.value
+  keycloak_postgres_password             = data.aws_ssm_parameter.keycloak_postgres_password.value
+  enable_google_login                    = var.enable_google_login
+  google_identity_provider_client_id     = var.enable_google_login ? data.aws_ssm_parameter.google_identity_provider_client_id[0].value : ""
+  google_identity_provider_client_secret = var.enable_google_login ? data.aws_ssm_parameter.google_identity_provider_client_secret[0].value : ""
+  enable_test_user                       = var.enable_test_user
+  test_user_password                     = var.enable_test_user ? data.aws_ssm_parameter.test_user_password[0].value : ""
```

Comment:
do these params need to be added to the toolchain module in environment-local as well?

Reply:
yeah, but this already doesn't work anyways so I figured it would be better to wait until someone picks up the ticket to get local environment working again

Reply:
we kind of keep kicking that can down the road and making it a bigger task for whoever takes it on